### ISSUE TITLE
envVars fails when the value has '=' querystring

### ISSUE THREAD

**Issue by jechav** [google-github-actions/deploy-cloud-functions#111]
### TL;DR
When i have a env variable whit querystrings like this one `MONGO_URL=mongodb+srv://<user>:<pass>@cluster0.xxx.mongodb.net/hubspotTaskCreator?retryWrites=true&w=majority` fails because the split  [here](https://github.com/google-github-actions/deploy-cloud-functions/blob/e95a0e3fd9239d725c4e439ab209581a5f1cb760/src/cloudFunction.ts#L167)

**Expected behavior**
Should cut the string only using the first '=', `pair.substring(pair.indexOf('=')+1)`

**Observed behavior**
Right now the result value is this one `mongodb+srv://<user>:<pass>@cluster0.xxx.mongodb.net/hubspotTaskCreator?retryWrites`

**Comment by robcowie** [comment]
I experienced the same problem yesterday and traced it to the same part of the code.

For better or worse I imagine it is common to provide a DB URI to a cloud function as an env var so this is likely to crop up frequently.

**Comment by bharathkkb** [comment]
Thanks for the report @jechav @robcowie 
Would passing in via `env_vars_file` be an option?

**Comment by robcowie** [comment]
I'm currently passing a value from Github secrets to the cloud function env var. As far as I'm aware I cannot access the envs (in this case `${{ secrets.xyz }}`) in an env vars file.

My vote would be to split only on the first occurrence of the delimiter in each key/val pair. It's more intuitive behaviour.

**Comment by bharathkkb** [comment]
@robcowie gotcha and that suggestion sgtm. Happy to review a PR too if you get to it first.

**Comment by robcowie** [comment]
@bharathkkb Small PR with one possible solution. I must admit I haven't yet run the tests as I'm not sure how to. Hoping they'll be run automatically on the PR!

**Comment by bharathkkb** [comment]
fixed by https://github.com/google-github-actions/deploy-cloud-functions/pull/122
